crates/autopilot/src/arguments.rs

```diff
@@ -82,6 +82,15 @@ pub struct Arguments {
     )]
     pub token_quality_cache_expiry: Duration,
 
+    /// How often to run background bad token detection maintenance task.
```

Comment:
We already have a tonne of arguments.
Let's try to make it work without this once since the only requirement is that it runs more frequently than `--token-quality-cache-expiry`.

Reply:
Used this value: `cache_expiry.div(10).max(Duration::from_secs(30))`